### DISCUSSION TITLE
Calling MediaElement's Handler.DisconnectHandler() is mandatory in Android??

### DISCUSSION THREAD

**Discussion by bhavanesh2001** [CommunityToolkit/Maui#2274] [Q&A]
I'm currently trying to use Media element in popup pages(community toolkit popup & and other one's) to display a video. While closing the popup/or navigating back from media element screen is it necessary to call DisconnectHandler()? if I only just call Stop() method, I am facing some weird issue. From this point if try to kill the app , It wont get completely killed ?? it always opens at the same page i tried to kill the app. not form the app start. I had restart mobile again to launch app from launch screen.

Note* I observed this in android

**Comment by ghost** [comment]
Related to this issue about [Windows + handler disconnect](https://github.com/CommunityToolkit/Maui/issues/1690) and also take a look at [this answer about dispose not needed anymore](https://github.com/CommunityToolkit/Maui/discussions/1663#discussioncomment-8730079).

Mandatory to call DisconnectHandler? [Looking at the example, it clearly shows you have to call it, when navigatedFrom](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Views/MediaElement/MediaElementPage.xaml.cs).
